### PR TITLE
Added lib dependency for psycopg2

### PR DESCRIPTION
**Pull request by wittejm**
I updated the Dockerfile and the readme, but only with an instruction for Linux and not for Mac or Windows, because I don't know those environments..

Also fixed the (typo) numbering on other install instructions.

### CODE REVIEW

**Comment by NickSchimek** [comment]
Thanks @wittejm. This looks good. I believe for the mac the `libpq` headers are installed if `postgresql` is installed using `homebrew`. If your on a mac you may or may not need to run `export LDFLAGS="-L/usr/local/opt/openssl/lib”` after installing `postgresql` via `brew install postgresql`.

**Comment by NickSchimek** [comment]
For devs wanting to use windows I would suggest that they use the docker image or partition ubuntu onto their hard drive. There's been to many issues trying to get Windows users on-boarded, and I don't think it's worth the headache at this point.

File: README.md [hunk 1]
```diff
@@ -56,10 +56,29 @@ A project to automate expunging qualifying criminal records.  This project is do
 	package manager which also automatically creates and manages virtual 
 	environments.
 
+4. Install additional libraries needed for running the backend natively or with pipenv:
+
+    * **Mac**
+
+      ```
+      brew install postgres
```
Comment (NickSchimek) [inline README.md]: Brew command for postgres is actually `brew install postgresql` <- this is only needed if the user doesn't have postgres installed already. More information regarding the install can be found here https://wiki.postgresql.org/wiki/Homebrew

Comment (erikjasso) [reply README.md]: I just want to point out that for Windows users who want to use Ubuntu there is an Ubuntu app on the Microsoft Store that runs Ubuntu natively in Windows, no need to partition your disk and install yourself.

Installing Ubuntu app:
https://docs.microsoft.com/en-us/windows/wsl/install-win10

Installing Docker Desktop on Windows:
https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly

It does take some PowerShell-fu that was new to me coming from Linux but once Ubuntu and Docker Desktop are configured, you can run the Ubuntu app from the Windows menu or type 'bash' from PowerShell to get a bash prompt (or use a bash-friendly console like ConEmu)

Comment (wittejm) [reply README.md]: Woops I will fix that line add that link for more information re the homebrew install. 

Given @erikjasso 's comment, should we just point that out at the top of the README and remove instructions for installing components on Windows?

**Pull request merged**